libs/market-list/src/lib/components/select-market.tsx

```diff
@@ -1,60 +1,64 @@
 import { useQuery } from '@apollo/client';
 import { t, volumePrefix } from '@vegaprotocol/react-helpers';
-import { Interval } from '@vegaprotocol/types';
 import {
   Dialog,
   Intent,
+  Link,
   Popover,
   RotatingArrow,
 } from '@vegaprotocol/ui-toolkit';
 import classNames from 'classnames';
 import { useMemo, useState } from 'react';
-import { MARKET_LIST_QUERY } from '../markets-data-provider';
 import type { Column } from './select-market-columns';
 import {
   columnHeadersPositionMarkets,
   columnsPositionMarkets,
 } from './select-market-columns';
 import { columnHeaders } from './select-market-columns';
 import { columns } from './select-market-columns';
-import type { MarketList } from '../__generated__';
+import type { MarketList_markets } from '../__generated__';
 import { useVegaWallet } from '@vegaprotocol/wallet';
 import type { Positions } from '@vegaprotocol/positions';
 import { POSITIONS_QUERY } from '@vegaprotocol/positions';
-import { mapDataToMarketList } from '../utils/market-utils';
 import {
   SelectMarketTableHeader,
   SelectMarketTableRow,
 } from './select-market-table';
+import { useMarketList } from '../markets-data-provider';
 
 export const SelectMarketLandingTable = ({
   data,
   onSelect,
 }: {
-  data: MarketList | undefined;
+  data: MarketList_markets[] | undefined;
   onSelect: (id: string) => void;
 }) => {
-  const marketList = data && mapDataToMarketList(data);
+  const textDecorationClassName = `px-8 underline font-sans leading-9 font-bold tracking-tight decoration-solid text-ui light:hover:text-black/80 dark:hover:text-white/80 text-black dark:text-white`;
```

Comment:
question (non-blocking): Do we really need all these links styles here. Why isn't the link component sufficient?

Reply:
we don't need this anymore! removed it in recent commit.